specs/rearrangements.yaml

```diff
@@ -139,6 +147,10 @@ fields:
     type: integer
     doc: FWR4 end coord in sequence (transferred from germline)
   -
+    name: duplicate_count
+    type: integer
+    doc: Count of duplicate sequences for this sequence
```

Comment:
Pre-processing tools like VDJPipe and pRESTO will collapse duplicate reads into a single entry then provide a count. For VDJPipe, it has to be exact length and nucleotide; I think that might be same criteria for pRESTO. This can greatly reduce the size of the sequence file, and speeds up alignment. That count needs to be carried forward to any subsequent analysis.

Adaptive Biotech has something similar with their template count.